### DISCUSSION TITLE
I published my first simple multiplayer game to the itch.io (Min0taur: PvP)

### DISCUSSION THREAD

**Discussion by ImperaEtConquer** [AlmasB/FXGL#1062] [Show and tell]
Finally did that. Never actually realized that finishing the game could be that hard.
Not quite happy with the final result, but at least it is playable and working.

Please kindly follow the link below and check if you're interested:
https://0a3r.itch.io/min0taur-pvp

Turn-based multiplayer game for two players with simple mechanics.

**Comment by AlmasB** [comment]
Very well done and congratulations on releasing your first game!

Downloading now :)

**Reply by AlmasB** [reply]
Looks great! https://twitter.com/AlmasBaim/status/1400802422931476481
